Mage/src/main/java/mage/game/draft/DraftImpl.java

```diff
@@ -284,6 +327,12 @@ public boolean addPick(UUID playerId, UUID cardId, Set<UUID> hiddenCards) {
         }
         return !player.isPicking();
     }
+    
+    @Override
+    public void setBoosterLoaded(UUID playerId) {
+        DraftPlayer player = players.get(playerId);
+        player.setBoosterLoaded();
```

Comment:
When coding cards we would usually check if the player we get for a given ID null.

Is player guaranteed to not be null?

Reply:
It's guaranteed to be null. You can look at `addPick` above this, it also doesn't check for null.